lib/resources/example_json/r4_examples_goal.rb

```diff
@@ -0,0 +1,180 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+    R4_GOAL ||= {
```

Comment:
R4_GOAL: This is being used for both provider and patient access. Add a separate example for Patient access since it will have lesser response

Reply:
As we don't have any changes in response if it is patient access. So I guss we can use the same one